cypress-accessibility-checker/src/commands.js

```diff
@@ -133,13 +134,19 @@ Cypress.Commands.add(
         }
       })
       .then((result) => {
-        if ((result !== 0 && shouldFail === undefined) || shouldFail === true) {
-          assert.fail(
-            'accessibility-checker: See previous logs for accessibility violation data'
-          );
-        }
         return cy.wrap(result, { log: false });
       });
+
+    const message =
+      'accessibility-checker: See previous logs for accessibility violation data';
+
+    if (shouldFail === true) {
+      assert.fail(message);
```

Comment:
I think this could be improved as well, but for now left it as is. I'm not sure what scenario i would want to force my test to fail, but I am sure someone might need to

Reply:
`shouldFail` probably needs to be documented better, I think it is causing some misunderstanding for what it's intended to be used for.

By default, the `assertA11yCompliance()` will fail a test if it finds violations.  If the developer doesn't want to fail the test (for example they want to see all the violations they have first), then they can send `false` as a value for `shouldFail`.  If no value is sent in or `true` is set, then it will behave the same way.  False will skip failing the test.

I believe 143/144 can be removed.  The if check you have, in combination with the new `.should()` syntax should cover what was being done previously.

Maybe the `shouldFail` should be changed to something like `shouldFailOnError`?

Reply:
That makes sense. I updated this to reflect that functionality. I also made an update to the README>